instat/dlgPlot.designer.vb

```diff
@@ -66,7 +66,7 @@ Partial Class dlgPlot
         '
         'cmdLineOptions
         '
-        Me.cmdLineOptions.Location = New System.Drawing.Point(10, 228)
+        Me.cmdLineOptions.Location = New System.Drawing.Point(10, 198)

```

Comment:
Switched the two command buttons around. Other dialogs tend to have
Dialog-name Options
Plot Options

This dialog had it the other way